### DISCUSSION TITLE
How to start with the fingerprintjs-master on own webserver

### DISCUSSION THREAD

**Discussion by merlincom** [fingerprintjs/fingerprintjs#859] [Q&A]
Hallo guys
I downloaded the file fingerprintjs-master.zip to my web directory and unzipped it to fingerprint-master. Which script do I have to include in the web page or which order file from fingerprint-master can I call in the browser?  Many thanks for hints. Marcel

**Comment by Finesse** [comment]
The Git archive has no files that are ready to run in browser, it has only source files.

Try the installation guide: https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/api.md#installation
